sharing_code.html

```diff
@@ -4,42 +4,80 @@
-            </li>
-        </ul>
-    </div>
+    <header>
+        <nav class="navbar has-shadow has-background-primary is-fixed-top">
+            <div class="navbar-brand">
+                <a class="navbar-item" href="index.html"><img src="assets/img/codecademy_logo.png" alt="The codecademy logo" class="logo"></a>
+                <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navbar-items">
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                    <span aria-hidden="true"></span>
+                </a>
+            </div>
+            <div class="navbar-menu" id="navbar-items">
+                <div class="navbar-end">
+                    <a class="navbar-item" href="./index.html">Home</a>
+                    <a class="navbar-item" href="./staff.html">Server Staff</a>
+                    <a class="navbar-item" href="./contact.html">Contact Us</a>
+                    <a class="navbar-item" href="./sharing_code.html">Sharing Code</a>
+                </div> 
+            </div>
+        </nav>
+    </header>
+    <main class="has-text-centered">
+        <div class="container p-2">
+            <h1 class="is-size-1 mt-6">How to share code effectively:</h1>
+            <p class="mb-6">A (non-exhaustive) list of some of the preferred ways to share code.</p>
+        </div> 
+        <section class="container has-background-white mb-6 p-6">
+            <div>
+                <h2 class="is-size-2 p-0">Small Snippets</h2>
+
+                <p>If sharing a single line of code, wrap it in a pair of backticks (`):</p>
+                <img src="./assets/img/singleline.png">
 
+                <p>For a larger chunk of code, wrap it in three backticks (`) with the name of the language:</p>
+                <img src="./assets/img/snippet.png">
+                <p>This has a number of benefits such as improved readability and syntax highlighting.</p>
+            </div>
+        </section>
+        <section class="container has-background-white mb-6 p-6">
+            <div>
+                <h2 class="is-size-2 p-0">Larger Code Files</h2>
+                <p>If you need to send a larger code snippet or a whole file, please avoid uploading it directly as a file (as file uploads are not permitted for security reasons) and instead use one of the options listed below:</p>
+                <section class="container">
+                    <h3 class="is-size-3">Exporting Code from Codecademy</h3>
+                    <p>If you need to export code directly from the Codecademy workspace, you can click on the Tools header on the navbar and export your code to a gist, which you can then share a link to:</p>
+                    <img src="./assets/img/cc_sharing.png">
+                </section>
+                <hr>
+                <section class="container">
+                    <h3 class="is-size-3">Sharing off-platform code</h3>
```

Comment:
I'm 50/50 on whether these headers should have numbers, like 
"1. Exporting Code from Codecademy" and 
"2. Sharing off-platform code".

I'm curious if you and @Victoria-DR think they look better with or without.